src/closure.md

```diff
@@ -183,13 +183,13 @@ The callbacks are defined by implementing the [`Delegate`] trait. The
 [`InferBorrowKind`][ibk] type implements `Delegate` and keeps a map that
 records for each upvar which mode of borrow was required. The modes of borrow
```

Comment:
It would be nice to link to the docs I think, especially because the name overlaps.
```suggestion
records for each upvar which mode of borrow was required. The possible [`BorrowKind`]s are
```
and then a link to https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/enum.BorrowKind.html